server/src/main/java/org/elasticsearch/cluster/DiffableUtils.java

```diff
@@ -402,6 +402,25 @@ public void writeTo(StreamOutput out) throws IOException {
         }
     }
 
+    private static <K, T> Map.Entry<K, T> mapEntry(K key, T newValue) {
```

Comment:
Just a quick fix here to enable `null` values as we can have them for settings. The default map entries don't allow null values.